docs/reference/modules/node.asciidoc

```diff
@@ -206,6 +213,67 @@ To create a dedicated data node, set:
 node.roles: [ data ]
 ----
 
+Data nodes can be more specialised in order to allow for tiered storage deployments.
+A collection of data nodes with the same specialised role is called a
+<<modules-tiers, data tier>>.
+
+IMPORTANT: If a node has a `data tier` role, it cannot *also*
+be a generic `data` node.
+
+The available <<modules-tiers, data tier>> roles are the following:
+
+[[data-content-node]]
+==== [x-pack]#Content data node#
+
+Content data nodes accommodate user-created content. They enable operations like CRUD,
+search and aggregations.
```

Comment:
I think introducing more abstract terms could potentially complicate things further here. I believe the product catalog would usually be manually introduced in the system (ie. user created) as opposed to being machine generated (like logs and metrics).

Reply:
I wonder if it would be clearer if we talk about "content" by exemplifying it as opposed to using the content origin? 

eg. Content data nodes store the documents that back/support application, website, and enterprise search. The value of the data in a content node remains relatively constant, and the performance requirements aren't tied to the age of the data.